internals/secrethub/init.go

```diff
@@ -128,11 +126,8 @@ func (cmd *InitCommand) Run() error {
 	case InitModeSetupCode:
 		setupCode := cmd.setupCode
 		if setupCode == "" {
-			var err error
-			setupCode, err = ui.AskAndValidate(cmd.io, "What is your setup code?\n", 3, api.ValidateSetupCode)
-			if err != nil {
-				return err
-			}
```

Comment:
Now only the additional prompt is gone, but we can also delist/hide setup codes entirely from the `secrethub init` options, so remove this too: https://github.com/secrethub/secrethub-cli/blob/465b8da80e898acfef90fb1653f4c2571204d354/internals/secrethub/init.go#L99

Reply:
Sure. That works together nicely with https://github.com/secrethub/secrethub-cli/pull/342